### PR TITLE
Add unit test for ParentGeometryTest

### PR DESCRIPTION
**Pull request by dongheeleeme**
I found that we could have a unit test for ParentGeometryTest. So, I've added a unit test for this class in this pr. Thanks!

### CODE REVIEW

File: contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+  fun `centerY method`() {
+    assertThat(parentGeometry.centerY()).isEqualTo(YInt(20))
+  }
+
+  @Test
+  fun `padding method`() {
+    assertThat(parentGeometry.padding().top).isEqualTo(10)
+    assertThat(parentGeometry.padding().right).isEqualTo(10)
+    assertThat(parentGeometry.padding().bottom).isEqualTo(10)
+    assertThat(parentGeometry.padding().left).isEqualTo(10)
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt:86]: Maybe we could put a different padding on each side. Tests wouldn't fail if say all padding methods were returning `top` for instance.

File: contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  private val paddingConfig = {
+    Rect().apply {
+      top = 10
+      right = 10
+      left = 10
+      bottom = 10
+    }
+  }
+  private val mockWidthConfig = SizeConfig(available = 40, lambda = lambda)
+  private val mockHeightConfig = SizeConfig(available = 40, lambda = lambda)
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt:38]: No need for the `mock` prefix. And I would use a different value for height and width.

File: contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt (hunk 1 above)
Comment (dongheeleeme) [reply contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt]: Thanks, make sense. Update in 90036f8

File: contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt (hunk 2 above)
Comment (dongheeleeme) [reply contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt]: Updated in 90036f8

File: contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+      top = 10
+      right = 20
+      bottom = 30
+      left = 40
+    }
+  }
+  private val parentGeometry = ParentGeometry(
+    SizeConfig(available = 120, lambda = { it }),
+    SizeConfig(available = 60, lambda = { it }),
+    paddingConfig
```
Comment (oldergod) [inline contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt:39]: name parameters 🙏

**Review by oldergod — APPROVED** [review]

File: contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt (hunk 3 above)
Comment (dongheeleeme) [reply contour/src/test/kotlin/com/squareup/contour/wrappers/ParentGeometryTest.kt]: Updated in 2add20e

**Review by oldergod — APPROVED** [review]

**Comment by oldergod** [comment]
Thanks

**Pull request merged**